blog/_posts/2017-07-25-team-berlindiamonds-part-one.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Creating our own Plugin, Yeeey!
+layout: post
+created_at: Tue Jul 25 2017
+permalink: blog/2017-07-25-team-berlindiamonds
+current: blog
+author: Kaja
+twitter: AlizeNero
+categories:
+- news
+---
+
+![image](../img/blog/2017/2017-07-firstday-confetti.jpg)
```

Comment:
Could you please add an image credit for this? (i.e. who took the picture, like you did for the second picture)